recipes/openssl/3.x.x/conanfile.py

```diff
@@ -282,7 +279,7 @@ def _targets(self):
             "Windows-x86-Visual Studio": "VC-WIN32",
             "Windows-x86_64-Visual Studio": "VC-WIN64A",
             "Windows-armv7-Visual Studio": "VC-WIN32-ARM",
-            "Windows-armv8-Visual Studio": "VC-WIN64-ARM",
```

Comment:
The `VC-WIN64-ARM` is for "Windows OneCore" as per https://openssl-library.org/policies/platforms/index.html, and is not the one intended when targetting desktop Windows arm64.